src/Microsoft.AspNetCore.Mvc.Core/Internal/DefaultApplicationModelProvider.cs

```diff
@@ -218,16 +223,24 @@ protected virtual PropertyModel CreatePropertyModel(PropertyInfo propertyInfo)
 
             var attributes = propertyInfo.GetCustomAttributes(inherit: true);
 
-
+            // BindingInfo for properties can be either specified by decorating the property with binding spcecific attributes
+            // of if there's binder model metadata associated with the property.
```

Comment:
"binder model metadata" sounds a bit specific (note your typo), almost like you're talking just about buddy types. Suggest
``` c#
// ... with binding-specific attributes.
// ModelMetadata also adds information from the property's type and any configured IBindingMetadataProvider.
```